src/System.AppContext/src/System/AppContext.cs

```diff
@@ -44,6 +45,19 @@ public static object GetData(string name)
             return null;
         }
 
+        public static event UnhandledExceptionEventHandler UnhandledException
```

Comment:
> how we implement some of the stuff like UnhandledException

UnhandledException handling needs to live in S.P.CoreLib.

Most methods on AppContext today are tightly coupled with the runtime, host or toolchain. You cannot really move it out for S.P.CoreLib for CoreCLR unless you introduce Augments class in CoreLib that most of the methods will forward to. The fact that it is separate for PN is false sense of layering.